models/quickbooks__balance_sheet.sql

```diff
@@ -0,0 +1,30 @@
+with general_ledger_by_period as (
+    select *
+    from {{ref('quickbooks__general_ledger_by_period')}}
+    where financial_statement_helper = 'balance_sheet'
+
+), final as (
+    select
+        {% if target.type == 'bigquery' %}
+            cast(format_date("%Y", date_year) as {{ dbt_utils.type_string() }}) as calendar_year,
+            cast(format_date("%b", period_last_day) as {{ dbt_utils.type_string() }}) as calendar_month,
+
+        {% else %}
```

Comment:
why do we do this?

Reply:
Reverted and changed to the `period_first_day as calendar_date` to show year and month within the single field opposed to the previous unnecessary logic.